tests/publiccloud/storage_perf.pm

```diff
@@ -0,0 +1,144 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Use FIO tool to run storage performance test
+#
+# Maintainer: Jose Lausuch <jalausuch@suse.com>
+
+use base "publiccloud::basetest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use repo_tools 'generate_version';
+
+use constant NUMJOBS => 4;
+use constant IODEPTH => 4;
+
+
+sub run {
+    my ($self) = @_;
+    my $reg_code = get_required_var('SCC_REGCODE');
+    my $runtime = get_var('RUNTIME',  300);
```

Comment:
If these variables are not globally used, I would add a namespace like `PUBLIC_CLOUD_FIO_`

Reply:
done